### ISSUE TITLE
Expected end of input

### ISSUE THREAD

**Issue by TitanNano** [dat2/ecmascript#4]
What ever I'm trying to parse, I'm getting:

```
Parse error at line: 1, column: 1
Unexpected `c`
Expected `end of input`
```

where `c` is the first character in the content.

I simply copied the example to get started...

**Comment by dat2** [comment]
Hi! Thanks for using my project! I took a “documentation first” approach and the documentation is slightly ahead of the actual code :sweat_smile:.

I am working on a PR (see #1) to implement all expressions, and remove the statements so that I can make them in another PR, but I flip flopped a bit on the AST format and unit testing / integration testing and that’s why it’s been open for so long :disappointed:.

Unfortunately due to the huge scope of the ecmascript language it will be a while before the full language is supported! I will merge in this PR with some documentation (in a couple days hopefully) on what language features are currently supported to unblock you though!

**Comment by TitanNano** [comment]
@dat2 I understand, in this case I will just wait a little until your project is ready to be tried out.

**Comment by dat2** [comment]
@TitanNano I just published a new version of the library on crates.io! The parser doesn't cover the entire language yet and probably will still have confusing error messages like these, but you should be able to get quite a few expressions parsed!

**Comment by TitanNano** [comment]
yes `var` declarations work now, I was able to test that. for property access I now get a new error.

```
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: ParseFloatError { kind: Invalid }', libcore/result.rs:1009:5
```

```JavaScript
var object = { test: 2, value: 'ab' }; object.test;
```
